easybuild/easyconfigs/v/vsc-mympirun/vsc-mympirun-4.0.2.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'vsc-mympirun'
+version = '4.0.2'
+
+homepage = 'https://github.com/hpcugent/vsc-mympirun'
+description = """VSC-tools is a set of Python libraries and scripts that are commonly used within HPC-UGent."""
+
+# we build this to work with every python version
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['a6ef244a80e29ddb1197c1d1bfac6f7b9a0092abb44c4cff789db42e27a3e1ad']
+
+builddependencies = [('vsc-install', '0.10.26')]
+dependencies = [('vsc-base', '2.5.8')]
+
+osdependencies = ['python-setuptools']
+
+# we ship something in bin/fake
+modextrapaths = {'PATH': 'bin/fake'}
+
+# don't check for import of vsc.mympirun, since that may fail if vsc-base is picked up from the OS
+options = {'modulename': 'vsc'}
+
+pyshortver = '.'.join(SYS_PYTHON_VERSION.split('.')[:2])
+sanity_check_paths = {
+    'files': ['bin/mympirun', 'bin/mympisanity'],
+    'dirs': ['bin/fake', 'lib/python%s/site-packages' % pyshortver],
```

Comment:
don't we have constants for this?

Reply:
`%(pyver)s` & co only work when Python is included as a (proper) dependency, not when it's a system dep